lib/elixir/lib/keyword.ex

```diff
@@ -1316,4 +1316,87 @@ defmodule Keyword do
   def size(keywords) do
     length(keywords)
   end
+
+  @doc """
+  Filters `keywords` i.e. returns a keyword list containing
+  only the elements from `keywords` for which the function
+  `fun` returns a truthy value.
```

Comment:
We should make this one as terse as Reject's docs, ex:

```
Returns a keyword list containing only the elements from `keywords`
for which the function `fun` returns a truthy value.
```

Reply:
The current wording was based on `Enum.filter/2`'s:

> Filters the `enumerable`, i.e. returns only those elements
  for which `fun` returns a truthy value.

I do like your new terser wording better.